tests/vrf_test.py

```diff
@@ -267,3 +267,11 @@ def test_invalid_vrf_name(self):
         result = runner.invoke(config.config.commands["vrf"].commands["del"], ["VrF10"], obj=obj)
         assert result.exit_code != 0
         assert expected_output in result.output
+
+        expected_output = """\
+Error: 'vrf_name' length should not exceed 16 characters
+"""
```

Comment:
@arfeigin please move it to a separate python file and then import. Please use an examples from other tests

Reply:
I followed the convention used in vrf_test.py file, can you point me to an example please?